src/components/program/ProgramHeader.jsx

```diff
@@ -1,17 +1,58 @@
 import React, { useContext } from 'react';
-
-import { Container } from '@edx/paragon';
+import { Parallax } from 'react-parallax';
+import { breakpoints } from '@edx/paragon';
 import { ProgramContext } from './ProgramContextProvider';
 
 const ProgramHeader = () => {
-  const { program: { title } } = useContext(ProgramContext);
+  const { program: { subjects, marketingHook } } = useContext(ProgramContext);
+
+  let isMobileWindow = true;
+
+  // Use the first subject as the primary subject
+  const primarySubject = subjects?.length > 0 ? subjects[0] : '';
+  const subjectSlug = primarySubject?.slug ? primarySubject?.slug.toLowerCase() : '';
+
+  const handleResize = () => {
+    const isMobileWindowRefresh = window.matchMedia(`(max-width: ${breakpoints.small.maxWidth}px)`).matches;
+    if (isMobileWindow !== isMobileWindowRefresh) {
+      isMobileWindow = !isMobileWindow;
+    }
+  };
+
+  const getHeaderBackgroundImage = () => {
```

Comment:
This has been replicated the same as to [prospectus](https://github.com/edx/prospectus/blob/0af9c2932e2816e49eb71c0aa1eb22b409436152/src/components/Program/ProgramHeader.jsx#L46).